ui/package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can we double check if we can add package.json and package-lock.json to the root pathway so everything is immediately usable after checking out the repo including running UI commands rather than having to cd into a nested directory?